glide.lock

```diff
@@ -1,10 +1,12 @@
-hash: f3e767e8346d32c0a86b9c441a9f5a580a4da1e4e9b201cffc7e2b313149bc5d
-updated: 2017-11-14T15:40:02.556527845+05:30
+hash: 9be3af04ee1417f46551109c7ed120aa3bb93a56a471144a7be35464d180f775
+updated: 2017-11-22T14:55:30.531845973+05:30
 imports:
 - name: github.com/davecgh/go-spew
   version: 5215b55f46b2b919f50a1df0eaa5886afe4e3b3d
   subpackages:
   - spew
+- name: github.com/fsnotify/fsnotify
```

Comment:
about the vendor update how did you make the changes? Because for the kubeval repo I also see the README.md and all irrelevant files in there?